syntax = "proto3";

option go_package = "aetherlink-gs-backend/proto";

message AetherLinkTelemetry {
  float roll_deg = 1;
  float pitch_deg = 2;
  float yaw_deg = 3;
  double latitude_deg = 4;
  double longitude_deg = 5;
  float relative_altitude_m = 6;
}
